sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIOTest.java

```diff
@@ -296,6 +297,7 @@ public void testReadingEmptyTable() throws Exception {
 
   /** Tests reading all rows from a table. */
   @Test
+  @Category(NeedsRunner.class)
```

Comment:
I'm slightly on the fence about these. I think this is an accurate use of the category, yet I feel like there might be another interesting subcategory that we should create. Let's move the discussion to dev list and come up with a solution there.